app/frontend/src/features/profile/view/ProfilePage.tsx

```diff
@@ -93,10 +92,17 @@ export default function ProfilePage() {
           <div className={classes.root}>
             <Overview user={user} setIsRequesting={setIsRequesting} />
             <Card className={classes.detailsCard} id={REQUEST_ID}>
-              <TabContext value={currentTab}>
+              <TabContext value={tab}>
                 <TabBar
-                  value={currentTab}
-                  setValue={setCurrentTab}
+                  value={tab}
+                  setValue={(newTab) =>
+                    history.push(
+                      routeToUser({
+                        username: user.username,
+                        tab: newTab,
+                      })
+                    )
```

Comment:
> * /user/home redirects to /user/aapeli/home

This use case doesn't seem to be supported I think because of this. Could we support it somehow without needing the redirect, considering `/user/about` works fine to take you to your user's "about" section currently?

Reply:
Not sure if I understand what I should do with this. 
I think this is what you want: 

- current page: `/user`
- click on _about_ or _home_
- url changes to: `/user/about` or `/user/home`
- while with my code it will change to `/user/username/about` and `/user/username/home`

Maybe a follow issue can further look into optimising the routes?

Reply:
Yes, the not needing redirect is the less important part, it's more the lack of the consistency this introduces if we push this out, so I'd prefer that we fix it here rather than pushing this out and leave for later.

Reply:
Ok, no problem. I just don't immediately know how can I prevent 'about' and 'home' to be seen as a username for the urls:
`/user/about` and `/user/home` when path is configured with two optional parameters like `/user/:username?/:tab?`

We can maybe cover this tonight in the dev meeting?

Reply:
![image](https://user-images.githubusercontent.com/4201102/118644650-5284f500-b7de-11eb-900b-b58539f0042c.png)

Reply:
One approach could be to have to base routes. One of the profile of the current user and one of viewing profiles of other users:

```
/me
/me/(edit/)about
/me/(edit/)home
```

and
```
/user/username
/user/username/about
/user/username/home
```

('me' can be anything else of course such as 'profile')

Reply:
I see, I missed part of the discussion near the beginning but I think we agreed (?) what you proposed there is probably the best alternative that makes sense and more resilient to change than hardcoding a list of special words that can't be usernames (which I'm almost certain we'll keep on changing as we keep on adding tabs to that list...)

Reply:
Ooops.. I thought we would go for the option where we would look on the profilepage (frontend) if the username parameter (the first parameter) is either "home" or "about". If that's the case we use the following parameters:
username: undefined
tab: home (or about)

But I now I read your comment, I believe you would actually vote for having to seperate base routes?

Reply:
Will look at separating routes in #1272